pkgs/by-name/pd/pdftowrite/package.nix

```
@@ -0,0 +1,80 @@
+{
+  lib,
+  python3Packages,
+  fetchFromGitHub,
+  makeWrapper,
+  versionCheckHook,
+  nix-update-script,
+
+  # shared
+  gzip,
+  # pdftowrite
+  poppler_utils,
```

```suggestion
  poppler-utils,
```
poppler_utils is an alias and shouldn't be used in nixpkgs.